### DISCUSSION TITLE
GIT HUB ACCOUNT

### DISCUSSION THREAD

**Discussion by codingboy3014** [community/community#145794] [New to GitHub]
### Select Topic Area

General

### GitHub Feature Area

Actions

### Body

I WANT 3 GIT HUB ACCOUNTS IN ONE EMAIL BUT I CAN NOT CREATE THAT WHAT I DO

**Comment by codingboy3014** [comment]
THE STEPS TO CREATE OR ANY SOLUTION

**Comment by the-bipu** [comment]
i think you can create multiple organization with single email but not the account. try using different emails.

**Comment by CyberSphinxxx** [comment]
Hi @codingboy3014.

Unfortunately, GitHub doesn’t allow you to create multiple accounts directly tied to the same email address. Each GitHub account must have a unique email address associated with it. However, here are some workarounds you can try:

### 1. **Use Gmail Aliases**
- If you use Gmail, you can add a `+` to your email address. For example:
  - `example@gmail.com` can become:
    - `example+account1@gmail.com`
    - `example+account2@gmail.com`
    - `example+account3@gmail.com`
- These will all go to the same inbox, but GitHub sees them as different emails.

### 2. **Use Other Emails**
- If you can, create extra email accounts. Services like Gmail or Outlook make it quick and easy to set up new ones.

- Do you really need three accounts? Instead of managing multiple accounts, you could:
  - Use **organizations** on GitHub to separate your projects.
  - Create different repositories for different purposes within one account.

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @codingboy3014 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by ghostinhershell** [comment]
Hi @codingboy3014, 

Thanks so much for reaching out in our community. I recommend checking out our Docs article on [Managing multiple accounts](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/managing-multiple-accounts) for more information on how you can use multiple accounts on GitHub. Thanks!

**Reply by codingboy3014** [reply]
I want those for m youtube and personal use
